themes/default/layouts/partials/header.html

```diff
@@ -123,9 +123,9 @@
 
                         <li class="mobile-menu-item"><a href="#">Learn</a></li>
                         <li class="mobile-menu-subitem"><a href="{{ relref . "/blog" }}">Blog</a></li>
-                        <li class="mobile-menu-subitem"><a href="{{ relref . "/resources" }}">Resources</a></li>
+                        <li class="mobile-menu-subitem"><a href="{{ relref . "/learn" }}">Learn Pulumi</a></li>
                         <li class="mobile-menu-subitem"><a href="{{ relref . "/docs/tutorials" }}">Tutorials</a></li>
-                        <li class="mobile-menu-subitem"><a href="{{ relref . "/resources#upcoming" }}">Workshops</a></li>
+                        <li class="mobile-menu-subitem"><a href="{{ relref . "/resources/#upcoming" }}">Events & Workshops</a></li>
```

Comment:
do we want to change this to all? or do you think upcoming still makes sense?

Reply:
I think upcoming still makes sense for now as those are where we want people to convert. 

I do want to follow up with some layout formats to the Events & Workshops page as I think we can do a lot of things we can do to improve the experience there.

Reply:
k, sounds good